sbcl 2.1.1: Remove debug output

equill opened the pull request:
Remove some debug output from the `patchPhase` that seems to serve no
lasting purpose.

###### Motivation for this change

As suggested by @SuperSandro2000 when reviewing my commit adding sbcl-2.1.1, I've removed a few lines whose purpose only seemed to be to confirm that the right version was being compiled.


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ * ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ * ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
   - n/a - no tests for this
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ * ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
   - n/a - no relevant documentation
- [ * ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

equill committed:
sbcl 2.1.1: Remove debug output

Remove some debug output from the patchPhase that seems to serve no
lasting purpose.

r-rmcgibbo commented:
Result of `nixpkgs-review pr 113132` at 75e514d0 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/c7de69c9dd9619a02c1a3b81929cd877">sbcl_2_1_1</a></li></li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`buildPhase` should probably contain `runHook preBuild` and `runHook postBuild`.

Near pkgs/development/compilers/sbcl/2.1.1.nix:77:3:
```
   |
77 |   buildPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/development/compilers/sbcl/2.1.1.nix:86:3:
```
   |
86 |   installPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/patch-phase.md">patch-phase</a>

`patchPhase` should not be overridden, use `postPatch` instead.

Near pkgs/development/compilers/sbcl/2.1.1.nix:23:3:
```
   |
23 |   patchPhase = ''
   |   ^
```
</li>
  </ul>
</details>

r-rmcgibbo commented:
Result of `nixpkgs-review pr 113132` at 75e514d0 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/76037a1a628ca19bff39ad3616bb4952">sbcl_2_1_1</a></li></li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`buildPhase` should probably contain `runHook preBuild` and `runHook postBuild`.

Near pkgs/development/compilers/sbcl/2.1.1.nix:77:3:
```
   |
77 |   buildPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/development/compilers/sbcl/2.1.1.nix:86:3:
```
   |
86 |   installPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/patch-phase.md">patch-phase</a>

`patchPhase` should not be overridden, use `postPatch` instead.

Near pkgs/development/compilers/sbcl/2.1.1.nix:23:3:
```
   |
23 |   patchPhase = ''
   |   ^
```
</li>
  </ul>
</details>

SuperSandro2000 commented on pkgs/development/compilers/sbcl/2.1.1.nix:
```diff
@@ -21,10 +21,6 @@ stdenv.mkDerivation rec {
   buildInputs = [texinfo];
 
   patchPhase = ''
```
```suggestion
  postPatch = ''
```

otherwise we can't add patches via the patches variable.

equill replied:
Thanks for the suggestion. I've implemented that, and added the missing hooks - the effort's trivial, it reduces complaints from the bot, and the reasoning looks sound.
It still compiles and runs, and the resulting compiler cleanly builds a non-trivial project, so it certainly doesn't seem to have hurt anything.

equill committed:
sbcl-2.0.9: move patchPhase to postPatch

Rename the `patchPhase` variable to `postPatch`, to enable adding
patches via the `patches` variable.

equill committed:
sbcl-2.1.1: add missing phase-hooks

Per suggestion of the build-bot, add `runHook preBuild` and `runHook
postBuild` to `buildPhase`, and add `runHook preInstall` and `runHook
postInstall` to `installPhase`.

SuperSandro2000 commented on pkgs/development/compilers/sbcl/2.1.1.nix:
```diff
@@ -20,11 +20,7 @@ stdenv.mkDerivation rec {
-
-    pwd
-
+  postpatch = ''
```
```suggestion
  postPatch = ''
```

SuperSandro2000 reviewed (changes requested):
also please resolve the merge conflict.

equill commented:
Resolving the merge-commits got messy, so I've rolled the changes up into a single commit on a fresh branch and created https://github.com/NixOS/nixpkgs/pull/113390 to replace this one.
